homeassistant/components/netgear_lte/__init__.py

```diff
@@ -28,6 +28,15 @@
 DOMAIN = 'netgear_lte'
 DATA_KEY = 'netgear_lte'
 
+EVENT_SMS = 'netgear_lte_sms'
```

Comment:
`netgear_lte.sms`?

Reply:
Uh, I am not sure. I see examples with a dot and examples with an underscore. The [docs](https://developers.home-assistant.io/docs/en/dev_101_events.html) do not even use a prefix.

Let me summon @balloob and I will update the docs and this PR with whatever he says.

Reply:
There is another page with a suggestion: https://developers.home-assistant.io/docs/en/creating_component_code_review.html#6-event-names

If we find a solution/decision here this page should be updated.

Reply:
The dot makes me think of a hierarchy while the underscore makes me think of the flat namespace that event names really are.

Both are fine with me though, I don't care at all except about consistency :)

Reply:
I prefer underscore